frontend/store/auctions.ts

```diff
@@ -26,9 +27,11 @@ interface State {
     auctionStorage: Record<string, AuctionTransaction>;
     takeEventStorage: Record<string, TakeEvent[]>;
     areAuctionsFetching: boolean;
+    isSelectedAuctionFetching: boolean;
     areTakeEventsFetching: boolean;
     isBidding: boolean;
     error: string | null;
+    auctionErrors: Record<string, string>;
```

Comment:
The type suppose to be `Record<string, string | undefined>` since some keys may be missing